test/regression/targets/sc-12024.cc

```diff
@@ -38,8 +38,8 @@ TEST_CASE("C++ API: Truncated values (ch12024)", "[cppapi][sparse]") {
     q.set_data_buffer("Z", z);
     REQUIRE_THROWS_WITH(
         q.submit(),
-        "[TileDB::Dimension] Error: Coordinate -682.75 is out of domain "
-        "bounds [-682.73999, 929.42999] on dimension 'Z'");
+        "[TileDB::Dimension] Error: Dimension: Coordinate -682.75 is out of "
+        "domain bounds [-682.73999, 929.42999] on dimension 'Z'");
```

Comment:
Nothing to do for this PR, but the format of our error messages has lots of room for improvement.